browser/src/Plugins/Api/shell-env.d.ts

```diff
@@ -0,0 +1 @@
+declare module "shell-env"
```

Comment:
Ah ya, I was looking to see if there was a DefinitelyTyped definition for this (`npm install @types/shell-env`), but unfortunately there isn't - so we'll have to go with this plan! Thanks for figuring this out.

Reply:
I tried adding some typing to the module declaration but not exactly sure how that's done and  could quickly figure this out from the docs